Fits/Electronic Attack Frigates/[ADV-30] Kitsune.md

```diff
@@ -0,0 +1,42 @@
+# Kitsune
+
+Advanced EWAR Frigate
+
+## Description
+
+Utilizes Electronic Counter Measures (ECM) to disrupt hostile DPS or Logistics. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary.
+
+If you know what the enemy is going to be fielding on grid, use the list below to pick the appropriate ECM modules since they are stronger against their specific targets.
+
+Ladar - Minmatar, Angel Cartel.
+Radar - Amarr, Blood Raiders, Sansha's Nation, Triglavian Collective.
+Magnometric - Gallente, ORE, Sisters of Eve.
+Gravimetric - Caldari, Guristas, Mordu's Legion.
+
+## Fit
+```
+[Kitsune, [ADV-30] Kitsune]
```

Comment:
I definitely think T2 ECM should be a requirement for the Kitsune, it's way better

Reply:
> I definitely think T2 ECM should be a requirement for the Kitsune, it's way better

The primary issue was cost. it takes it from a 43.3mil fit to a 74.8mil fit with refits, and the actual % jam chance increase is really not there for a 2x price increase. Example on the Absolute Honor CFI (1sigamp) with mindlinked info bursts:

T2:

1 jam - 22%
2 jam - 39.2%
3 jam - 52.5%
4 jam - 63%

Scoped:

1 jam - 19.8%
2 jam - 35.7%
3 jam - 48.4%
4 jam - 58.6%

So it's basically a 2.2% - 4.4% increase in % chance from scoped to T2 depending on how many jams you have on them, which just didn't eel like it was worth nearly doubling the price for.

Reply:
Huh? T2 ECM mods are like 2m each, this should be an 8m difference

Reply:
> Huh? T2 ECM mods are like 2m each, this should be an 8m difference

You need the refits as well. It's 4x sets of 4 mods.